solidity/ecdsa/deploy/06_deploy_wallet_registry_governance.ts

```diff
@@ -7,7 +7,8 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
 
   const WalletRegistry = await deployments.get("WalletRegistry")
 
-  const GOVERNANCE_DELAY = 604800 // 1 week
+  // 1 week for mainnet, 1 second for other test and dev networks.
+  const GOVERNANCE_DELAY = hre.network.name === "mainnet" ? 604800 : 1
```

Comment:
Ah, wait - I think we should have another value here for Hardhat network. Maybe `60` sec only for Goerli?

Reply:
Why? Aren't we interested in effectively getting rid of the governance delay for all non-mainnet networks?

Reply:
Ah yeah, tests. Will change

Reply:
See b838911